clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/CloudLoggingProcessor.java

```diff
@@ -69,35 +64,22 @@ public AuditLogRequest process(AuditLogRequest auditLogRequest) throws LogProces
                               .preservingProtoFieldNames()
                               .print(auditLogRequest.getPayload()),
                           new TypeReference<Map<String, ?>>() {})))
-              .setLogName(getLogNameFromLogType(auditLogRequest.getType()))
               .setLabels(auditLogRequest.getLabelsMap())
-              .setResource(MonitoredResource.newBuilder(MONITORED_RESOURCE_TYPE).build())
+              .setOperation(
+                  Operation.of(
+                      auditLogRequest.getOperation().getId(),
+                      auditLogRequest.getOperation().getProducer()))
+              .setTimestamp(
+                  Instant.ofEpochSecond(
+                      auditLogRequest.getTimestamp().getSeconds(),
+                      auditLogRequest.getTimestamp().getNanos()))
```

Comment:
Are these types incompatible? Couldn't we just do:

```
.setOperation(auditLogRequest.getOperation())
.setTimestamp(auditLogRequest.getTimestamp())
```

Reply:
They are incompatible:
setOperation needs Operation whereas auditLogRequest.getOperation() is LogEntryOperation 
setTimestamp needs Instant whereas auditLogRequest.getTimestamp() is google.protobuf.Timestamp

Reply:
Ack, makes sense.

For the timestamp, [using this](https://github.com/googleapis/java-logging/blob/059d62acd81190489122490cc072d0d0cb5e8cd8/google-cloud-logging/src/main/java/com/google/cloud/logging/JavaTimeConversions.java#L51) may be helpful.

Reply:
Seems it is not visible outside of that package, so keeping the way it is.